Password.java

```diff
@@ -0,0 +1,58 @@
+public class Password {
+	private static final String ALGORITHM = "AES";
+	private static final String INIT_VECTOR = "ZwNnZZXfhanqzUtZV6gQQg==";
+	private static final String KEY = "upW8BpO89Qwilpr4XG4j3Q==";
+	private static final String PADDING = "/CBC/PKCS5Padding";
+	private Key key;
+
+	public Key getKey(){
+		return this.key;
+	}
+	
+	public String getINIT_VECTOR(){
+		return this.INIT_VECTOR;
+	}
+
+
+	public String getAlgorithm(){
+		return this.key;
+	}
+
+
+	public Password() throws Exception {
+		byte[] encodedKey = Base64.decodeBase64(KEY);
+		key = new SecretKeySpec(encodedKey, 0, encodedKey.length, ALGORITHM);
+	}
+
+	public String encrypt(String plaintext) throws Exception {
+		return encrypt(Base64.decodeBase64(INIT_VECTOR), plaintext);
+	}
+
+	public String encrypt(byte[] iv, String plaintext) throws Exception {
```

Comment:
it is hard to understand what "iv" parameter stands for, the name should explain itself